### ISSUE TITLE
[Feature Request]Please move the icon to the center of the screen along with transparency option.

### ISSUE THREAD

**Issue by veeraragavan-n** [gabm/TapAndTurn#45]
Please move the icon to the center of the screen along with the option to adjust the transparency. Searching which corner to activate rotation is far less intuitive than just touching the center.

**Comment by licaon-kter** [comment]
There is no center option (was requested before #41), but the corner is fixed and predictable, as you set it up in Settings.

Also you can already set transparency in Settings,

/Close this

**Comment by veeraragavan-n** [comment]
As a former developer, I'm more of an end user these days. So I understand
technical descriptions of color in the form of RGB + Alpha, my father,
mother or grandparents cannot or won't be able to figure out such things
either. It would be helpful for perpetually novice end-users like them if
you can change the string to something self-explanatory like transparency.

**Comment by licaon-kter** [comment]
Just make a PR :grinning:

**Comment by gabm** [comment]
I agree with you, it's not specially user friendly! good suggestion...

If you feel like I'd be happy to include your contribution!

**Comment by veeraragavan-n** [comment]
Thanks. I'd love to contribute but don't see a donate button anywhere yet.

On Dec 2, 2017 6:30 PM, "gabm" <notifications@github.com> wrote:

I agree with you, it's not specially user friendly! good suggestion...

If you feel like I'd be happy to include your contribution!

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/gabm/TapAndTurn/issues/45#issuecomment-348690674>, or mute
the thread

**Comment by gabm** [comment]
well I hoped more for contributing code ;)

**Comment by veeraragavan-n** [comment]
Sure. Code would be even better.

**Comment by ghost** [comment]
More options for where to put the icon would be really nice, perhaps even an option to set the exact x/y position per rotation mode?
